pontoon/base/utils.py

```diff
@@ -214,13 +215,19 @@ def get_download_content(slug, code, part):
             get_object_or_404(Resource, project__slug=slug, path=relative_path)
         ]
 
-    for resource in resources:
-        # Get locale file
-        locale_prefixes = (
-            project.repositories.filter(permalink_prefix__contains="{locale_code}")
+    if project.configuration_file:
+        locale_prefixes = project.repositories.values_list(
+            "permalink_prefix", flat=True
+        )
+    else:
+        locale_prefixes = list(
+            project.repositories.filter(Q(permalink_prefix__contains="{locale_code}"))
             .values_list("permalink_prefix", flat=True)
             .distinct()
         )
```

Comment:
Can we only make the distinction (based on presence of `project.configuration_file`) for the QS, and then apply `values_list()` and `distinct()` in one place only?

Also, can we omit that `Q()` wrapper?